curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-map-data-structure.english.md

```diff
@@ -58,8 +58,9 @@ tests:
 ```js
 var Map = function() {
   this.collection = {};
-  // change code below this line
-  // change code above this line
+  // Only change code below this line
+  
+  // Only change code above this line
```

Comment:
This file has two areas where users need to change code, and comment says "**only** change code below/above this line". Seems a little inaccurate since it's not the only place to change code. There's probably some other files like this as well - I would be fine leaving it, users can figure that out. Just thought I would point it out.

Reply:
Related to your other comment above.  The only time you should see the "Only change code below this line" and "Only change code above this line", is when there is only one place where the user should be changing something.  The other version without the "Only" was for challenges that had more than one location where we expected changes.

I will do another review, to validate I did not mix up any of these and create a separate commit to fix them.

Reply:
> We're in the coding interview section, users know how to do things at this point - I don't think we need to tell them where to put their code. I feel like so many of these would be fine without comments. 

At this point, I would rather not remove any comments as long as they are the correct comment.